### PR TITLE
[NZPMC-070] select from userquizzes

### PR DESCRIPTION
**Pull request by ruby-oujo**
**Issue**

Query multiple userQuiz, Query one userQuiz, Add page for viewing multiple userQuiz

**Solution**

Implemented multiple additional buttons for start and continue / continue to start / start to continue with  multiple colours 

**Risk**

Made changes to welcome page and exam page and also apollo mutations  

**Reviewers**

@scorpionknifes  @Will-ZYS  @lanxdrey19

### CODE REVIEW

File: src/views/Exam.vue [hunk 1]
```diff
@@ -138,23 +138,19 @@ export default {
...
-            query: UserQuizzesQuery,
+            query: UserQuizQuery,
+            variables() {
+                return { quizID: this.$route.params.quizId }
+            },
             update: (data) => {
-                return data.userQuizzes[0]
+                return data.userQuiz
             },
+            fetchPolicy: 'no-cache',
```
Comment (ruby-oujo) [inline src/views/Exam.vue:148]: There is an issue with apollo cache @Will-ZYS  please have a look @scorpionknifes  gave up. Remove line to remove hot fix.

Comment (Will-ZYS) [reply src/views/Exam.vue]: fixed

**Review by Will-ZYS — APPROVED** [review]
LGTM

**Comment by matthewaptaylor** [comment]
LGTM

**Review by matthewaptaylor — APPROVED** [review]
LGTM

**Pull request merged**